### PR TITLE
v1.0.1

### PR DESCRIPTION
**Pull request by aravindvnair99**
Summary:

- Switched from HTTP to HTTPS links
- Updated `gitignore`
- Updated package name
- Fixed incorrect package version
- Updated dependencies
- Fixed missing labels in issue templates
- Open links in a new tab instead of the current window
- Updated link of `GithubCorner`
- Compressed images
- Resized images to `400px*400px`
- Refactored repo
- Make contributing guidelines clearer
- Added missing info
- Fixed PULL_REQUEST_TEMPLATE
- Updated docs

### CODE REVIEW

File: src/components/json/worldData.js [hunk 1]
```diff
@@ -66,9 +68,42 @@ const data = [
 		bio: "Progate Community Intern",
 		url: "https://dsc.community.dev/bhagalpur-college-of-engineering/",
 		chapterName: "Indian Institute of Information Technology, Bhagalpur",
-		github: "https://github.com/helios027",
+		github: "http://github.com/helios027",
```
Comment (abhinavsri360) [inline src/components/json/worldData.js:71]: Shouldn't this be `https`?

Comment (aravindvnair99) [reply src/components/json/worldData.js:71]: @abhinavsri360 Yes, that change was made later in `main`. I'll update this PR. One moment.

**Review by abhinavsri360 — APPROVED** [review]
Looks good to me.
P.S. - Too many binary file changes.

**Comment by aravindvnair99** [comment]
> Looks good to me.
> P.S. - Too many binary file changes.

Dependencies 😅

**Pull request merged**